samples/java/play-authenticate-usage/build.sbt

```diff
@@ -2,24 +2,25 @@ import play.PlayJava
 
 name := "play-authenticate-usage"
 
-scalaVersion := "2.11.2"
+scalaVersion := "2.11.5"
 
 version := "1.0-SNAPSHOT"
 
 val appDependencies = Seq(
-  "be.objectify"  %% "deadbolt-java"     % "2.3.2",
+  "be.objectify" %% "deadbolt-java" % "2.4.0-SNAPSHOT",
   // Comment the next line for local development of the Play Authentication core:
-  "com.feth"      %% "play-authenticate" % "0.6.8",
+  "com.feth"      %% "play-authenticate" % "0.7.0",
   "postgresql"    %  "postgresql"        % "9.1-901-1.jdbc4",
   javaCore,
   cache,
   javaWs,
-  javaJdbc,
-  javaEbean,
-  "org.webjars" %% "webjars-play" % "2.3.0",
-  "org.webjars" % "bootstrap" % "3.2.0"
+  "com.typesafe.play" % "play-jdbc_2.11" % "2.4.0-M3",
+  "org.webjars" %% "webjars-play" % "2.4.0-SNAPSHOT",
```

Comment:
I don't think we need to include this dependency anymore - I think play already includes webjars anyway.